metadata-ingestion/src/datahub/ingestion/source/unity/connection_test.py

```diff
@@ -9,14 +12,48 @@
 from datahub.ingestion.source.unity.proxy import UnityCatalogApiProxy
 from datahub.ingestion.source.unity.report import UnityCatalogReport
 
+try:
+    from azure.identity import DefaultAzureCredential
+
+    AZURE_AVAILABLE = True
+except ImportError:
+    AZURE_AVAILABLE = False
```

Comment:
this doesn't look right

let's just depend on azure identity in the unity source, regardless of whether or not it's absolutely required